lib/plugins/aws/provider.js

```diff
@@ -1816,7 +1817,21 @@ Object.defineProperties(
         const localTag = `${repositoryName}:${imageName}`;
         const remoteTag = `${repositoryUri}:${imageName}`;
 
-        const buildDockerArgs = ['build', '-t', localTag, '-f', pathToDockerfile, imagePath];
+        const buildArgsArr = [];
+        Object.keys(buildArgs).map((key) =>
```

Comment:
Let's introduce a test that will verify that introduced feature actually works properly - other related tests live here: https://github.com/serverless/serverless/blob/7601e26360226e77c4c8d631227a248e7a81afc2/test/unit/lib/plugins/aws/provider.test.js#L1094